src/createAction.js

```diff
@@ -8,12 +8,19 @@ export default function createAction(type, actionCreator, metaCreator) {
     : identity;
 
   return (...args) => {
-    const action = {
+    let action = {
```

Comment:
i guess i didn't have to. thought maybe i should since we modify action below, but we don't reassign.